Joplin_install_and_update.sh

```diff
@@ -140,17 +143,17 @@ else
 fi
 
 # Check if it's in the latest version
-if [[ -e ~/.joplin/VERSION ]] && [[ $(< ~/.joplin/VERSION) == "${RELEASE_VERSION}" ]]; then
+if [[ -e ${INSTALL_DIR}/joplin/VERSION ]] && [[ $(< ${INSTALL_DIR}/joplin/VERSION) == "${RELEASE_VERSION}" ]]; then
```

Comment:
```suggestion
if [[ -e ${INSTALL_DIR}/VERSION ]] && [[ $(< ${INSTALL_DIR}/VERSION) == "${RELEASE_VERSION}" ]]; then
```


With this change, `INSTALL_DIR` defaults to `~/.joplin` and, as such, Joplin will be installed to `~/.joplin/joplin`, instead of just `~/.joplin/`.

As this script is also used for updating, we may want to preserve the original `~/.joplin` install directory by default.